docs/source/deploying/docker.md

```diff
@@ -0,0 +1,55 @@
+# Using Docker
+
+<!--
+TODO: Add section on RHOAI officially supported images, once they exist
+!-->
+
+## Spyre base images
+
+Base images containing the driver stack for IBM Spyre accelerators are available from the [ibm-aiu](https://quay.io/repository/ibm-aiu/base?tab=tags) organization on Quay. This includes the `torch_sendnn` package, which is required for using torch with Spyre cards.
+
+:::{attention}
+These images contain an install of the `torch` package. The specific version installed is guaranteed to be compatible with `torch_sendnn`. Overwriting this install with a different version of `torch` may cause issues.
+:::
+
+## Using community built images
+
+Community maintained images are also [available on quay](https://quay.io/repository/ibm-aiu/vllm-spyre?tab=tags), the latest x86 build is `quay.io/ibm-aiu/vllm-spyre:latest.x86_64`.
+
+These images are provided as a reference and come with no support guarantees.
+
+## Building vLLM Spyre's Docker Image from Source
+
+You can build and run vLLM Spyre from source via the provided gh-file:docker/Dockerfile. To build vLLM Spyre:
```

Comment:
the `gh-file` shortcut didn't work for some reason

Reply:
fixed!